Common/Data/Custom/Accern.cs

```diff
@@ -0,0 +1,359 @@
+        {
+            var accernData = new Accern();
+            
+            var data = line.Split(',');
+
+            accernData.ArticleId = data[0];
+            accernData.StoryId = data[1];
+            accernData.Time = DateTime.ParseExact(data[2], "yyyy-MM-dd HH:mm:ss UTC", CultureInfo.InvariantCulture);
+            accernData.Name = data[3];
+            accernData.Symbol = Symbol.Create(data[4], SecurityType.Base, QuantConnect.Market.USA);
+            accernData.GlobalId = data[5];
+            accernData.EntityId = data[6];
+            accernData.Type = data[7];
+            accernData.Exchange = data[8];
+            accernData.Sector = data[9];
+            accernData.Industry = data[10];
+            accernData.Country = data[11];
+            accernData.Region = data[12];
+            accernData.Index = data[13];
+            accernData.Competitors = data[14];
+            accernData.Name2 = data[15];
+            accernData.Symbol2 = data[16];
+            accernData.GlobalId2 = data[17];
+            accernData.EntityId2 = data[18];
+            accernData.Type2 = data[19];
+            accernData.Exchange2 = data[20];
+            accernData.Sector2 = data[21];
+            accernData.Industry2 = data[22];
+            accernData.Country2 = data[23];
+            accernData.Region2 = data[24];
+            accernData.Index2 = data[25];
+            accernData.Competitors2 = data[26];
+            accernData.GroupName = data[27];
+            accernData.GroupType = data[28];
+            accernData.GroupName2 = data[29];
+            accernData.GroupType2 = data[30];
+            if(data[31] != "") accernData.StorySentiment = Convert.ToDecimal(data[31]);
+            accernData.StorySaturation = data[32];
+            if (data[33] != "") accernData.StoryVolume = Convert.ToInt32(data[33]);
+            if (data[34] != "") accernData.FirstMention = Convert.ToBoolean(data[34]);
+            accernData.ArticleType = data[35];
+            if (data[36] != "") accernData.ArticleSentiment = Convert.ToDecimal(data[36]);
+            if (data[37] != "") accernData.OverallSourceRank = Convert.ToInt32(data[37]);
+            if (data[38] != "") accernData.EventSourceRank = Convert.ToInt32(data[38]);
+            if (data[39] != "") accernData.EventSourceRank2 = Convert.ToInt32(data[39]);
+            if (data[40] != "") accernData.OverallAuthorRank = Convert.ToInt32(data[40]);
+            if (data[41] != "") accernData.EventAuthorRank = Convert.ToInt32(data[41]);
+            if (data[42] != "") accernData.EventAuthorRank2 = Convert.ToInt32(data[42]);
+            if (data[43] != "") accernData.OverallEventImpactScore = Convert.ToDecimal(data[43]);
+            if (data[44] != "") accernData.EventEntityImpactScore = Convert.ToInt32(data[44]);
+            if (data[45] != "") accernData.EventEntityImpactScore2 = Convert.ToInt32(data[45]);
```

Comment:
Nullable values can be a pain, but I think these should probably be nullables, otherwise how will the algorithm be able to discern between a real value of 0 vs it not being specified?